src/features/embalm/stepContent/hooks/useCreateSarcohpagus.ts

```diff
@@ -39,43 +42,64 @@ export function useCreateSarcophagus() {
   const { submitSarcophagus } = useSubmitSarcophagus();
 
   // TODO: Move this into its own hook and check all fields
-  // const canCreateSarcophagus = recipientState.publicKey !== '' && !!outerPublicKey && !!file; // TODO: Restore this line and remove line below when in-app create sarco flow is ready
-  const canCreateSarcophagus = !!outerPublicKey && !!file;
+  // TODO: render a message to the user about what data is missing
+  const canCreateSarcophagus = recipientState.publicKey !== '' && !!outerPublicKey && !!file;
```

Comment:
one thing to note, handling in a followup --- for the "generate new recipient public key" -- the public key only gets set in state *after* downloading the PDF of the keys (not after just generation). noting this down as a discussion point